lib/react_client/js_interop_helpers.dart

```diff
@@ -26,17 +27,20 @@ class _MissingJsMemberError extends Error {
       'cannot be used as expected. $message';
 }
 
+/// __Deprecated: use [js_util.getProperty] instead.
+///
 /// Returns the property at the given [_GetPropertyFn.key] on the
 /// specified JavaScript object [_GetPropertyFn.jsObj]
 ///
 /// Necessary because `external operator[]` isn't allowed on JS interop classes
 /// (see: https://github.com/dart-lang/sdk/issues/25053).
 ///
 /// __Defined in this package's React JS files.__
+@Deprecated('4.0.0')
```

Comment:
```suggestion
@Deprecated('6.0.0')
```

Reply:
These deprecations have been lumped into joe's ticket for master line un-deprecating of `shouldComponentUpdateWithContext` to be marked for 5.0.0 deprecation with alternative usage suggestions, this PR now completely removes them.